spec/lib/data_explorer_query_group_bookmarkable_spec.rb

```diff
@@ -0,0 +1,153 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe DataExplorerQueryGroupBookmarkable do
+  fab!(:admin_user) { Fabricate(:user) }
```

Comment:
```suggestion
  fab!(:admin_user) { Fabricate(:admin) }
```

Should this be `Fabricate(:admin)` instead?

Reply:
Ah yes, you're right.